### PR TITLE
How to Sync/update a forked repo from the original repo using Github Web UI and Git and submit your contributions to the project team

### PR DESCRIPTION
**Pull request by kimkimani**

### CODE REVIEW

**Comment by kimkimani** [comment]
@nadivgold Am using this pull request to write an article as I need some screenshots for the content

**Comment by kimkimani** [comment]
@nadivgold I hope it is okay.

**Comment by kimkimani** [comment]
@nadivgold I understand `wontfix` as issues currently not viable to solve or out of the scope of the project
I had the topic given a tick on slack with key takeaways as 
1. Update fork using GitHub web UI.
2. Update fork using Git Bash.
3. Submit contribution using Github Desktop.
4. Submit contributions using Git bush.
And also added the key labels used on the GitHub team project. As in labels used on the progress of the review of a pull request.
The topic is 80% done and I made an earlier pull request to be able to take some screenshots which as key to the topic.
I would love to understand if I can still continue with the topic on the subject @nadivgold  
Thank you.

**Comment by nadivgold** [comment]
@kimkimani Yes absolutely go for it. Tag was just an indicator for me that there is no article here at the moment so when I'm looking through the repo, I can more easily see articles that still need editing. Ping me when you have the article done and I'll remove the tag and get to it.

**Comment by kimkimani** [comment]
@nadivgold I  have updated the PR, and the content is ready for review.

**Comment by nadivgold** [comment]
> @nadivgold I have updated the PR, and the content is ready for review.

Great. I'll let @hectorkambow know that your article is ready to being the review process.

**Comment by hectorkambow** [comment]
deleted the label section - because that may change over time - and in our repo is more for internal use.

**Comment by hectorkambow** [comment]
@nadivgold Did a preliminary review on this (added the hero and front matter) - made some changes for flow - if you could give it just a tech review when you have a moment. 
🚀 
Thank you

**Comment by nadivgold** [comment]
I'm not an expert but everything seems good to me. It looks pretty accurate based on what I know about the topic.

File: articles/how-to-syn-and-update-a-forked-repo/index.md [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+-   **Push** is used to transfer commits made on local repository to GitHub remote repository. To push to GitHub you make a push request to update your local commits to your remote repository.
+-   **Branch** a feature branch or base branch. A feature branch typically helps you to isolate your changes from the other team as you make and test this changes. The base branch consists of merged feature branches.
+-   **Pull** is to notify the project team of the changes you have made from a pushed branch in your GitHub repository or basically requesting the owner of the repository to pull changes you made, thus called a pull request. A collaborator can be assigned to review your pull and propose any potential changes you need to make. Review changes are done with commits. Once changes and reviews are complete a team contributor or you (with access), can merge your branch to the base branch of the original repository.
```
Comment (nadivgold) [inline articles/how-to-syn-and-update-a-forked-repo/index.md:25]: Pulls are not Pull Requests. Change this to pull request and you'd be right. Pull is requesting the diff from a branch

File: articles/how-to-syn-and-update-a-forked-repo/index.md [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+-   **Merge** Merging is publishing a pull request after proposed reviews have been made. Merging is done by contributors with push access to the parent repository. You can also merge changes in the original repository to your forked repository.
```
Comment (nadivgold) [inline articles/how-to-syn-and-update-a-forked-repo/index.md:26]: Merge is merging two branches, whether they be PRs, feature branches, or from a pull from master to your local version.

**Comment by nadivgold** [comment]
Once these issues are addressed, it should be good for final review!

**Comment by kimkimani** [comment]
@nadivgold Addressed the issues

**Pull request merged**